pkg/boot/linux.go

```diff
@@ -26,6 +31,7 @@ type LinuxImage struct {
 	Initrd   io.ReaderAt
 	Cmdline  string
 	BootRank int
+	Syscall  string
```

Comment:
I'm not sure I like this API yet, but it should be an enum not a string

Reply:
I agree. It's a mess at present.

Reply:
it's now a bool. The string made no sense.